lib/stat_tracker.rb

```diff
@@ -61,13 +61,14 @@ def count_of_teams
   def best_offense
     @game_teams_manager.teams_sort_by_average_goal.last.team_name
   end
+
+  def worst_offense
+    @game_teams_manager.teams_sort_by_average_goal.first.team_name
```

Comment:
Nice job here John. Thanks for making sure all the tests are passing before pushing